suites/reef/rgw/tier-2_rgw_ms-archive.yaml

```diff
@@ -252,307 +256,92 @@ tests:
-        install_start_kafka: true
-        script-name: test_bucket_notifications.py
-        config-file-name: test_bucket_notification_kafka_broker_persistent_copy.yaml
-  - test:
-      name: notify on multisite replication create events with kafka_broker on arc site
-      desc: notify on multisite replication create events with kafka_broker on arc site
-      polarion-id: CEPH-83575922
-      module: sanity_rgw_multisite.py
-      clusters:
-        ceph-arc:
-          config:
-            run-on-rgw: true
-            extra-pkgs:
-              - wget https://download.oracle.com/java/17/latest/jdk-17_linux-x64_bin.rpm
-            install_start_kafka_archive: true
-            script-name: test_bucket_notifications.py
-            config-file-name: test_bucket_notification_kafka_broker_ms_replication_from_pri.yaml
-  - test:
-      name: notify on multisite replication delete events with kafka_broker on arc site
-      desc: notify on multisite replication delete events with kafka_broker on arc site
-      polarion-id: CEPH-83575922
-      module: sanity_rgw_multisite.py
-      clusters:
-        ceph-arc:
-          config:
-            run-on-rgw: true
-            script-name: test_bucket_notifications.py
-            config-file-name: test_bucket_notification_kafka_broker_archive_delete_replication_from_pri.yaml
-# adding the LC tests for object size filter before as they have failed in the end.
-  - test:
-      clusters:
-        ceph-pri:
-          config:
-            script-name: ../s3cmd/test_lifecycle_s3cmd.py
-            config-file-name: ../../s3cmd/multisite_configs/test_s3cmd_lifecycle_archive_objects_size_noncurrent.yaml
-      desc: Test LC on archive for objects size expiration
-      polarion-id: CEPH-83582000
-      module: sanity_rgw_multisite.py
-      name: Test LC on archive for objects size expiration
-  - test:
-      clusters:
-        ceph-pri:
-          config:
-            script-name: ../s3cmd/test_lifecycle_s3cmd.py
-            config-file-name: ../../s3cmd/multisite_configs/test_s3cmd_lifecycle_objects_size_noncurrent_local.yaml
-      desc: Test LC on active for objects size expiration
-      polarion-id: CEPH-83581990
-      module: sanity_rgw_multisite.py
-      name: Test LC on active for objects size expiration
-  - test:
-      clusters:
-        ceph-pri:
-          config:
-            set-env: true
-            script-name: user_create.py
-            config-file-name: tenanted_user.yaml
-            copy-user-info-to-site: ceph-arc
-      desc: create tenanted user
-      module: sanity_rgw_multisite.py
-      name: create tenanted user
-      polarion-id: CEPH-83575199
 
   - test:
       clusters:
         ceph-pri:
           config:
-            config-file-name: test_Mbuckets_with_Nobjects.yaml
-            script-name: test_Mbuckets_with_Nobjects.py
-            verify-io-on-site: [ "ceph-sec","ceph-arc" ]
-            timeout: 5000
```

Comment:
@viduship 
I think this timeout needs to be there as it is greater than 3600
@anrao19 what do you think?

Reply:
yes @hmaheswa

Reply:
addressed.